sdk/src/feature_set.rs

```diff
@@ -564,6 +568,7 @@ lazy_static! {
         (cap_accounts_data_size_per_block::id(), "cap the accounts data size per block #25517"),
         (preserve_rent_epoch_for_rent_exempt_accounts::id(), "preserve rent epoch for rent exempt accounts #26479"),
         (enable_bpf_loader_extend_program_data_ix::id(), "enable bpf upgradeable loader ExtendProgramData instruction #25234"),
+        (prevent_crediting_rent_paying_accounts::id(), "prevent crediting rent paying accounts #26606"),
```

Comment:
we are doing back and forth here; initially it was: `restrict_rent_paying_accounts_ttl_extension`.
@CriesofCarrots  asked to change it:
https://github.com/solana-labs/solana/pull/26606#discussion_r920775882

Reply:
Personally, I think it’s fine as-is, but to address Justin’s concern about lost nuance, it could be `prevent_crediting_accounts_that_end_rent_paying` or similar, I suppose.
I am opposed to the “lifetime” suggestion, as rust already has a definition of the word; I don’t think that “accounts lifetime extension” is that clear unless you already understand the problem, and I am disinclined to add new verbiage that ought to be explained, especially for functionality that is being made obsolete.

Reply:
done